regenerate.sh

```diff
@@ -31,8 +31,8 @@ echo "remove existing generated files"
 # intentionally generated by an older version of protoc-gen-go.
 rm -f $(find . -name '*.pb.go' | grep -v 'grpc_testingv3/testv3.pb.go')
 
-echo "go install github.com/golang/protobuf/protoc-gen-go"
-(cd test/tools && go install github.com/golang/protobuf/protoc-gen-go)
+echo "go install google.golang.org/protobuf/cmd/protoc-gen-go"
+(cd test/tools && go install google.golang.org/protobuf/cmd/protoc-gen-go)
```

Comment:
Why does the generated code still depend on `"github.com/golang/protobuf/proto"` with the codegen from `google.golang.org/protobuf`?

Reply:
See https://github.com/golang/protobuf/issues/1077 for why it originally existed. The dependency has since been removed, but won't be observable until we release v1.26.0.